### PR TITLE
Update to security-release-team@

### PR DESCRIPTION
**Pull request by lukehinds**
Resolves: #63 

Also updates links that now redirect

Signed-off-by: Luke Hinds <lhinds@redhat.com>

### CODE REVIEW

**Review by saschagrunert — APPROVED** [review]
Thank you @lukehinds!

File: security-release-process.md [hunk 1]
```diff
@@ -123,17 +123,17 @@ Their rotation will involve the following:
 
 #### SIG Release Roles
 
-Included on the [private Release Managers list](https://groups.google.com/a/kubernetes.io/forum/#!forum/release-managers-private)
+Included on the [private Release Managers list](https://groups.google.com/a/kubernetes.io/forum/#!forum/security-release-team)
 are the following members:
 
-- [Release Managers](https://git.k8s.io/sig-release/release-managers.md#release-managers)
+- [Release Managers](https://git.k8s.io/website/content/en/releases/release-managers.md)
```
Comment (tallclair) [inline security-release-process.md:129]: If we're linking to the website content, we might as well link to the actual website:
```suggestion
- [Release Managers](https://kubernetes.io/releases/release-managers/)
```

File: security-release-process.md [hunk 2]
```diff
@@ -123,17 +123,17 @@ Their rotation will involve the following:
...
 (manage build and release aspects when a security fix must be delivered)
-- [SIG Release Chairs](https://git.k8s.io/sig-release/release-managers.md#sig-release-chairs)
+- [SIG Release Chairs](https://git.k8s.io/website/content/en/releases/release-managers.md#chairs)
```
Comment (tallclair) [inline security-release-process.md:131]:
```suggestion
- [SIG Release Chairs](https://kubernetes.io/releases/release-managers/#chairs)
```

File: security-release-process.md [hunk 3]
```diff
@@ -123,17 +123,17 @@ Their rotation will involve the following:
...
 
 It is the responsibility of the [SIG Release Chairs](https://git.k8s.io/sig-release/release-managers.md#sig-release-chairs)
 to curate and maintain the various release role access controls across release cycles.
 
-The [Build Admins](https://git.k8s.io/sig-release/release-managers.md#build-admins)
+The [Build Admins](https://git.k8s.io/website/content/en/releases/release-managers.md#build-admins)
```
Comment (tallclair) [inline security-release-process.md:136]:
```suggestion
The [Build Admins](https://kubernetes.io/releases/release-managers/#build-admins)
```

File: security-release-process.md [hunk 4]
```diff
@@ -123,17 +123,17 @@ Their rotation will involve the following:
...
+Included on the [private Release Managers list](https://groups.google.com/a/kubernetes.io/forum/#!forum/security-release-team)
 are the following members:
 
-- [Release Managers](https://git.k8s.io/sig-release/release-managers.md#release-managers)
+- [Release Managers](https://git.k8s.io/website/content/en/releases/release-managers.md)
 (manage build and release aspects when a security fix must be delivered)
-- [SIG Release Chairs](https://git.k8s.io/sig-release/release-managers.md#sig-release-chairs)
+- [SIG Release Chairs](https://git.k8s.io/website/content/en/releases/release-managers.md#chairs)
 
 It is the responsibility of the [SIG Release Chairs](https://git.k8s.io/sig-release/release-managers.md#sig-release-chairs)
```
Comment (tallclair) [inline security-release-process.md:133]:
```suggestion
It is the responsibility of the [SIG Release Chairs](https://kubernetes.io/releases/release-managers/#chairs)
```

File: security-release-process.md [hunk 5]
```diff
@@ -123,17 +123,17 @@ Their rotation will involve the following:
...
 to curate and maintain the various release role access controls across release cycles.
 
-The [Build Admins](https://git.k8s.io/sig-release/release-managers.md#build-admins)
+The [Build Admins](https://git.k8s.io/website/content/en/releases/release-managers.md#build-admins)
 (Googlers with access to build/publish Kubernetes deb/rpm packages) are not
```
Comment (tallclair) [inline security-release-process.md:137]: Is this line still accurate?

**Review by justaugustus — COMMENTED** [review]
@lukehinds -- Ping to address Tim's review

**Comment by saschagrunert** [comment]
Hey @lukehinds, do we have any update on that?

File: security-release-process.md (hunk 5 above)
Comment (lukehinds) [reply security-release-process.md:137]: It resolves ok for me.

**Comment by lukehinds** [comment]
> Hey @lukehinds, do we have any update on that?

sorry for taking so long on this, accepted @tallclair recommendations

**Comment by tallclair** [comment]
/lgtm

Just need to fix the commit message to make the bot happy

**Comment by saschagrunert** [comment]
@lukehinds may I ask you to fix the `invalid-commit-message` label?

**Pull request closed without merging**